src/test/java/com/kodilla/ecommercee/domainTests/CartTestSuite.java

```diff
@@ -150,50 +134,10 @@ public void testRemoveProductFromCart() {
         cart.setProducts(retrievedProducts);
         savedCart = cartRepository.save(cart);
```

Comment:
ja bym to zrobiła tak, ale to tylko sugestia. nie jestem na 100% pewna. do przemyślenia i sprawdzenia przez kogoś jeszcze :)

(w linii od razu po //When znowu nie mogę komentować i ja bym zmieniła na
Optional<Cart> retrievedCart = cartRepository.findById(cart.getId());  )

```suggestion
        savedCart.setProducts(retrievedProducts);
        cartRepository.save(savedCart);
```